pkg/installer/custominstallconfig.go

```diff
@@ -54,16 +56,30 @@ func (m *manager) applyInstallConfigCustomisations(installConfig *installconfig.
 	}
 
 	aroManifests := &AROManifests{}
-	fileFetcher := &aroFileFetcher{directory: "/"}
-	aroManifestsExist, err := aroManifests.Load(fileFetcher)
+	manifestsFileFetcher := &aroFileFetcher{directory: "/"}
+	aroManifestsExist, err := aroManifests.Load(manifestsFileFetcher)
 	if err != nil {
-		m.log.Errorf("Error loading ARO manifests: %v", err)
+		err = fmt.Errorf("error loading ARO manifests: %w", err)
+		m.log.Error(err)
+		return nil, err
+	}
+
+	boundSaSigningKey := &tls.BoundSASigningKey{}
+	boundSaSigningKeyFileFetcher := &aroFileFetcher{directory: "."}
+	boundSaSigningKeyExists, err := boundSaSigningKey.Load(boundSaSigningKeyFileFetcher)
+	if err != nil {
+		err = fmt.Errorf("error loading boundSASigningKey: %w", err)
+		m.log.Error(err)
 		return nil, err
 	}
 
 	g := graph.Graph{}
 	g.Set(installConfig, image, clusterID, bootstrapLoggingConfig, dnsConfig, imageRegistryConfig)
 
+	if boundSaSigningKeyExists {
```

Comment:
Probably, even if doesn't exist, it can go in g.Set() above?

Reply:
I guess it wouldn't cause issues, we either provide 2 files or we don't.